tool/tsh/tsh.go

```diff
@@ -819,3 +825,73 @@ func onShow(cf *CLIConf) {
 
 	fmt.Printf("Fingerprint: %s\n", ssh.FingerprintSHA256(pub))
 }
+
+// onStatus command shows which proxy the user is logged into and metadata
+// about the certificate.
+func onStatus(cf *CLIConf) {
+	tc, err := makeClient(cf, true)
+	if err != nil {
+		utils.FatalError(err)
```

Comment:
I'm not sure about using FatalError everywhere , for example your defer won't be called and you will have a connection hanging. Can you wrap this into fucnction that returns proper error, and then fatal exit on the other layer.

Reply:
not the connection hanging, but connection state hanging

Reply:
it will be eventually GCed but still